src/m365/spo/commands/knowledgehub/knowledgehub-remove.ts

```diff
@@ -0,0 +1,132 @@
+          spoAdminUrl = _spoAdminUrl;
+          return this.getRequestDigest(spoAdminUrl);
+        })
+        .then((res: ContextInfo): Promise<string> => {
+          if (this.verbose) {
+            cmd.log(`Removing Knowledge Hub Site settings from your tenant`);
+          }
+
+          const requestOptions: any = {
+            url: `${spoAdminUrl}/_vti_bin/client.svc/ProcessQuery`,
+            headers: {
+              'X-RequestDigest': res.FormDigestValue
+            },
+            body: `<Request AddExpandoFieldTypeSuffix="true" SchemaVersion="15.0.0.0" LibraryVersion="16.0.0.0" ApplicationName="${config.applicationName}" xmlns="http://schemas.microsoft.com/sharepoint/clientquery/2009"><Actions><ObjectPath Id="29" ObjectPathId="28"/><Method Name="RemoveKnowledgeHubSite" Id="30" ObjectPathId="28"/></Actions><ObjectPaths><Constructor Id="28" TypeId="{268004ae-ef6b-4e9b-8425-127220d84719}"/></ObjectPaths></Request>`
+          };
+
+          return request.post(requestOptions);
+        })
+        .then((res: string): void => {
+          const json: ClientSvcResponse = JSON.parse(res);
+          const response: ClientSvcResponseContents = json[0];
+          if (response.ErrorInfo) {
+            cb(new CommandError(response.ErrorInfo.ErrorMessage));
+          }
+          else {
+            cmd.log(json[json.length - 1]);
+
+            if (this.verbose) {
+              cmd.log(vorpal.chalk.green('DONE'));
+            }
+            cb();
+          }
+        }, (err: any): void => this.handleRejectedPromise(err, cmd, cb));
+    };
+
+    if (args.options.confirm) {
+      if (this.debug) {
+        cmd.log('Confirmation bypassed by entering confirm option. Removing Knowledge Hub Site setting...');
+      }
+      removeKnowledgehub();
+    }
+    else {
+      cmd.prompt({
+        type: 'confirm',
+        name: 'continue',
+        default: false,
+        message: `Are you sure you want to remove Knowledge Hub Site from your tenant?`,
+      }, (result: { continue: boolean }): void => {
+        if (!result.continue) {
+          cb();
+        }
+        else {
+          removeKnowledgehub();
+        }
+      });
+    }
+  }
+
+  public options(): CommandOption[] {
+    const options: CommandOption[] = [
+      {
+        option: '--confirm',
+        description: 'Don\'t prompt for confirming removal of Knowledge Hub Site for your tenant'
+      }
+    ];
+
+    const parentOptions: CommandOption[] = super.options();
+    return options.concat(parentOptions);
+  }
+
+  public commandHelp(args: CommandArgs, log: (help: string) => void): void {
+    const chalk = vorpal.chalk;
+    log(vorpal.find(commands.KNOWLEDGEHUB_REMOVE).helpInformation());
+    log(
+      `  ${chalk.yellow('Important:')} to use this command you have to have permissions to access
+    the tenant admin site.
+
+  Examples:
+  
+    Removes the Knowledge Hub Site setting for your tenant
+      ${this.name}
```

Comment:
Let's prepend `m365` to code samples to make it easier for user to re-use them

Reply:
Sure.  Will make sure to add in future updates.👌